src/ax_interface/encodings.py

```diff
@@ -143,12 +143,12 @@ def size(self):
         return 4 + self.length + util.pad4(self.length)
 
     def __str__(self):
-        return self.string.decode('ascii')
+        return self.string.decode('utf-8')
```

Comment:
Fix it and add test case.

---
In reply to: [108312790](https://github.com/Azure/sonic-snmpagent/pull/19#discussion_r108312790) [](ancestors = 108312790)